sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -28,6 +28,15 @@ from form documents. It includes the following main functionalities:
 </dependency>
 ```
 [//]: # ({x-version-update-end})
+> Note: This version of the client library defaults to the `v2.1` version of the service.
+
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+|SDK version|Supported API version of service
+|-|-
+|3.0.0 | 2.0
+|3.0.1 | 2.0
```

Comment:
I know Java does constant patches to update Core dependency. Does that apply to the GA releases too?
if there are multiple you could do, `3.0.x` so that it is only one row